### DISCUSSION TITLE
直播间官方深色模式

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1949] [综合讨论 / General]
官方给直播间上深色模式了：https://t.bilibili.com/533553655363619175

**Comment by the1812** [comment]
官方那个有点奇怪啊... 很多边框都还是白色的

**Reply by Geeyun-JY3** [reply]
这好像是和 Bilibili Evolved 的兼容问题，可能和某个设置项有关（初步排查出是 `简化直播间` 和 `夜间模式`），实测是可以开启 Bilibili Evolved 的同时没有白色边框的。
